docs/documentation/platform/kms/encryption-strategies.mdx

```diff
@@ -0,0 +1,171 @@
+
+![HSM Illustration](/images/platform/kms/hsm/hsm-illustration.png)
+
+Using a hardware security module comes with the added benefit of having a secure and tamper-proof device to store your encryption keys. This ensures that your data is secure and protected from unauthorized access.
+
+<Warning>
+  All encryption keys used for cryptographic operations are stored within the HSM. This means that if the HSM is lost or destroyed, you will no longer be able to decrypt your data stored within Infisical. Most providers offer recovery options for HSM devices, which you should consider when setting up an HSM device.
+</Warning>
+
+Enabling HSM encryption has has a set of key benefits:
+1. **Root Key Wrapping**: The root KMS encryption key that is used to secure your Infisical instance, will be encrypted using the HSM device rather than encrypting with a software key.
+2. **FIPS 140-2/3 Compliance**: Using an HSM device ensures that your Infisical instance is FIPS 140-2 or FIPS 140-3 compliant. For FIPS 140-3, ensure that your HSM is FIPS 140-3 validated.
+
+### Requirements
+- Infisical instance with version that is equal to or greater than `v0.91.0`. 
+- HSM device from a provider such as [Thales Luna HSM](https://cpl.thalesgroup.com/encryption/data-protection-on-demand/services/luna-cloud-hsm), [AWS CloudHSM](https://aws.amazon.com/cloudhsm/), or others.
+
+
+### FIPS Compliance
+FIPS, also known as the Federal Information Processing Standard, is a set of standards that are used to accredit cryptographic modules. FIPS 140-2 and FIPS 140-3 are the two most common standards used for cryptographic modules. If your HSM uses FIPS 140-3 validated hardware, Infisical will automatically be FIPS 140-3 compliant. If your HSM uses FIPS 140-2 validated hardware, Infisical will be FIPS 140-2 compliant.
+
+HSM devices are especially useful for organizations that operate in regulated industries such as healthcare, finance, and government, where data security and compliance are of utmost importance.
+
+For organizations that work with US goverment agencies, FIPS compliance is almost always a requirement when dealing with sensitive information. FIPS compliance ensures that the cryptographic modules used by the organization meet the security requirements set by the US government.
+
+## Setup Instructions
+<Tabs>
+    <Tab title="Regular Encryption">
+        <Note>
+            Regular Encryption is the default encryption strategy used by Infisical KMS.
+        </Note>
+        <Steps>
+            <Step title="Setting up Regular Encryption">
+                To set up regular encryption, you need to define the `ENCRYPTION_KEY` environment variable. This key is used to encrypt all the data in Infisical KMS.
+
+                If this is not set correctly, your Infisical instance will not be able to start. You can read more about the [default instance configurations](/self-hosting/configuration/envars) here.
+            </Step>
+        </Steps>
+    </Tab>
```

Comment:
Suggestion - `regular` -> `standard`  and remove step (since it's only one it's just a gray box):

```
       <Tab title="Standard Encryption">
        <Note>
            Standard Encryption is the default encryption strategy used by Infisical KMS.
        </Note>
        To set up standard encryption, you need to define the `ENCRYPTION_KEY` environment variable. This key is used to
        encrypt all the data in your Infisical KMS.

        If this is not set correctly, your Infisical instance will not be able to start. You can read more about the
        [default instance configurations](/self-hosting/configuration/envars) here.
    </Tab>
```